mmv1/products/securitycenter/FolderSccBigQueryExports.yaml

```
@@ -0,0 +1,138 @@
+update_mask: true
+import_format:
+  - 'folders/{{folder}}/bigQueryExports/{{big_query_export_id}}'
+description: |
+  A Cloud Security Command Center (Cloud SCC) Big Query Export Config.
+  It represents exporting Security Command Center data, including assets, findings, and security marks
+  using gcloud scc bqexports
+  ~> **Note:** In order to use Cloud SCC resources, your organization must be enrolled
+  in [SCC Standard/Premium](https://cloud.google.com/security-command-center/docs/quickstart-security-command-center).
+  Without doing so, you may run into errors during resource creation.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/security-command-center/docs/how-to-analyze-findings-in-big-query'
+  api: 'https://cloud.google.com/security-command-center/docs/reference/rest/v1/folders.bigQueryExports'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'scc_folder_big_query_export_config_basic'
+    primary_resource_id: 'custom_big_query_export_config'
+    external_providers: ["random", "time"]
+    skip_test: true
+    vars:
+      big_query_export_id: 'my-export'
+      dataset: 'my-dataset'
+      dataset_id: 'my_dataset_id'
+      name: 'my-export'
+      folder_display_name: "folder-name"
+    test_env_vars:
+      org_id: :ORG_ID
+      project: :PROJECT_NAME
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: folder
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The folder where Cloud Security Command Center Big Query Export
+      Config lives in.
+  - !ruby/object:Api::Type::String
+    name: bigQueryExportId
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      This must be unique within the organization.
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    output: true
+    description: |
+      The resource name of this export, in the format
+      `projects/{{project}}/bigQueryExports/{{big_query_export_id}}`.
+      This field is provided in responses, and is ignored when provided in create requests.
+  - !ruby/object:Api::Type::String
+    name: description
+    description: |
+      The description of the notification config (max of 1024 characters).
+    validation: !ruby/object:Provider::Terraform::Validation
+      function: 'validation.StringLenBetween(0, 1024)'
+  - !ruby/object:Api::Type::String
+    name: dataset
+    description: |
+      The dataset to write findings' updates to.
+      Its format is "projects/[projectId]/datasets/[bigquery_dataset_id]".
+      BigQuery Dataset unique ID must contain only letters (a-z, A-Z), numbers (0-9), or underscores (_).
+  - !ruby/object:Api::Type::String
+    name: createTime
+    output: true
+    description: |
+      The time at which the BigQuery export was created. This field is set by the server and will be ignored if provided on export on creation.
```

Users can't provide it; let's just leave this off.

```suggestion
      The time at which the BigQuery export was created.
```

---

Thank you for the suggestion. Addressed.